tests/test_client.py

```diff
@@ -48,14 +48,15 @@
 class MockedClient(HttpClient, RateLimitHandler):
 
     def __init__(self, base_url, default_sleep_time=0.1, max_retries=1,
-                 status_force_list=HttpClient.DEFAULT_STATUS_FORCE_LIST,
-                 headers=HttpClient.DEFAULT_HEADERS, sleep_for_rate=False,
+                 extra_status_forcelist=[],
```

Comment:
Add some values to `extra_status_forcelist` to check those values are added to the list. You should check the values of the list to. The same with `extra_headers`. Update the dict with some other headers.